docs/en/reference/events.rst

```diff
@@ -164,7 +164,8 @@ the life-time of their registered entities.
    database insert operations. Generated primary key values are
    available in the postPersist event.
 -  preUpdate - The preUpdate event occurs before the database
-   update operations to entity data. It is not called for a DQL UPDATE statement.
+   update operations to entity data. It is not called for a DQL UPDATE statement
+   nor when the computed changeset is empty.
```

Comment:
To clarify: when reading the documentation as it is now it is not obvious that one of the preconditions is that one more or fields need to be modified in order for this to trigger.